File: Lab1/Lab1/Lab1/db/RAMRepository.cs
using Lab1.dto;
using Lab1Code.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab1.db
{
    class RAMRepository
    {
        public static List<RAMDto> findAll()
        {
            List<RAMDto> rAMs = new List<RAMDto>();
            rAMs.Add(new RAMDto (RAMTypes.SMOLLE, 100));
            rAMs.Add(new RAMDto(RAMTypes.BIG, 100));
            return rAMs;
        }
    }
}

File: Lab1/Lab1/lab1j/src/db/RAMRepository.java
package db;

import dto.RAMDto;
import dto.RAMTypes;

import java.util.LinkedList;
import java.util.List;

class RAMRepository
    {
        public static List<RAMDto> findAll()
        {
            List<RAMDto> rAMs = new LinkedList<>();
            rAMs.add(new RAMDto (RAMTypes.SMOLLE, 100));
            rAMs.add(new RAMDto(RAMTypes.BIG, 100));
            return rAMs;
        }
    }

